src/datumaro/plugins/data_formats/kaggle/base.py

```diff
@@ -125,7 +145,14 @@ def _load_items(self, ann_file: str, columns: Dict[str, Union[str, list]]):
 
         indices = {"media": df_fields.index(columns["media"])}
         if "label" in columns:
-            indices.update({"label": df_fields.index(columns["label"])})
+            label_columns = columns["label"]
+            if isinstance(columns["label"], list):
```

Comment:
This is a minor comment, but how about using `label_columns` instead of `columns["label"]`?

Reply:
Fix in [40ba7a4](https://github.com/openvinotoolkit/datumaro/pull/1607/commits/40ba7a4e8b43bd98d839b8ac259612bb51661237)